src/pca_report_library/utility/time.py

```diff
@@ -1,17 +1,22 @@
 """Convert times."""
-__all__ = ["time_to_string", "format_timedelta_to_HHMMSS"]
 
+__all__ = [
+    "time_to_string",
+    "format_timedelta_to_HHMMSS",
+]
 
-def time_to_string(secondsInput):
+
+def time_to_string(seconds_input):
     """Convert time in seconds to a string description."""
     string = ""
-    prior = False  # Used to ensure if the preceding period has a value that the following are all included.
-
+    # Used to ensure if the preceding period has a value that the
+    # following are all included.
+    prior = False
     # Creates the desired time calculations
-    days = secondsInput / (24 * 3600)
-    hours = (secondsInput % (24 * 3600)) // 3600
-    minutes = secondsInput % 3600 // 60
-    seconds = secondsInput % 60
+    days = seconds_input / (24 * 3600)
+    hours = (seconds_input % (24 * 3600)) // 3600
+    minutes = seconds_input % 3600 // 60
+    seconds = seconds_input % 60
```

Comment:
Something like this is less kludgy. Something similiar is done at https://github.com/cisagov/pca-report-library/blob/c100bdbf134b8f88330ac1f6c5042c121a564330/src/pca_report_library/utility/time.py#L54-L55

```suggestion
    days, seconds = divmod(seconds_input, 24 * 3600)
    hours, seconds = divmod(seconds, 3600)
    minutes, seconds = divmod(seconds, 60)
```